modules/end-user-guide/pages/adding-a-vscode-extension.adoc

```diff
@@ -27,42 +37,46 @@ The easiest way to add a Visual Studio Code extension is to add the extension to
     ]
   }
 ----
-. {prod-short} will automatically install the Red Hat Visual Studio Code Java extension in the {prod-short} workspace.
 
-Note: If the specified set of extensions IDs is not available in the {prod-short} registry, the workspace creation will not fail but the extensions will not be available.
+*NOTE* 
+If the specified set of extension ids isn't available in the {prod-short} registry, the workspace starts without the plug-in.
```

Comment:
If the specified set of extension ids isn't available in the {prod-short} registry, the workspace starts without the extension.